extract_baserom.py

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/python3
 
-import os
-import sys
 import struct
 from multiprocessing import Pool, cpu_count
+from pathlib import Path
 
 
-ROM_FILE_NAME = 'baserom.z64'
+ROM_FILE_NAME = Path('baseroms/gc-eu-mq-dbg/baserom_uncompressed.z64')
+SEGMENTS_PATH = Path('baseroms/gc-eu-mq-dbg/segments/')
 FILE_TABLE_OFFSET = 0x12F70
```

Comment:
slight semantics thing, but im wondering if `baserom_decompressed` makes more sense as a name. as in, the rom that was once compressed and then got de-compressed for extraction.

Either way the name doesnt really make that much sense for debug rom since its not compressed... but idk

Reply:
no matter what name we go with, we might want to switch to using a dash instead of an underscore, to match the rest of the rom names we've been going with